.github/workflows/format-code.yml

```diff
@@ -1,5 +1,5 @@
 name: Format code using Prettier
-on: [push, pull_request]
+on: workflow_dispatch
```

Comment:
You can also disable Actions to prevent automatic formatting in the meantime.

Reply:
Yeah, I know I just fogot to. I'm making it manual because WorldLanguages said he doesn't mind if it isn't automatic on Discord. I can undo that through.

Reply:
```suggestion
on: [push, pull_request]
```

Reply:
> Yeah, I know I just forgot to do it. I'm making it manual because WorldLanguages said he doesn't mind if it isn't automatic on Discord. I can undo that through.

He doesn't want the auto-commits, the check should still be there